sources/packages/backend/workflow/test/2022-2023/parttime-assessment/disbursements/parttime-assessment-disbursements.e2e-spec.ts

```diff
@@ -246,6 +280,295 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-disbursements`,
+          createFakeConfigureDisbursementPartTimeData(PROGRAM_YEAR);
+        configureDisbursementData.offeringStudyStartDate = getISODateOnlyString(
+          addDays(30),
+        );
+        configureDisbursementData.offeringStudyEndDate = getISODateOnlyString(
+          addDays(240),
+        );
+        configureDisbursementData.finalFederalAwardNetCSLPAmount =
+          testFinalAwardNetAmount.finalAwardNetAmount;
+        configureDisbursementData.finalFederalAwardNetCSGPAmount =
+          testFinalAwardNetAmount.finalAwardNetAmount;
+        configureDisbursementData.finalFederalAwardNetCSGDAmount =
+          testFinalAwardNetAmount.finalAwardNetAmount;
+        configureDisbursementData.finalFederalAwardNetCSPTAmount =
+          testFinalAwardNetAmount.finalAwardNetAmount;
+        configureDisbursementData.finalProvincialAwardNetBCAGAmount =
+          testFinalAwardNetAmount.finalAwardNetAmount;
+        configureDisbursementData.finalProvincialAwardNetSBSDAmount =
+          testFinalAwardNetAmount.finalAwardNetAmount;
+        // Act
+        const calculatedAssessment = await executePartTimeConfigureDisbursement(
+          configureDisbursementData,
+        );
+        // Assert
+        expect(
+          calculatedAssessment.variables.disbursementSchedules,
+        ).toStrictEqual([
+          {
+            disbursementDate: getISODateOnlyString(addDays(30)),
+            negotiatedExpiryDate: getISODateOnlyString(addDays(30)),
+            disbursements: [
+              {
+                awardEligibility: true,
+                valueAmount: testFinalAwardNetAmount.roundDownAwardNetAmount,
+                valueCode: "CSLP",
+                valueType: "Canada Loan",
+              },
+              {
+                awardEligibility: true,
+                valueAmount: testFinalAwardNetAmount.roundDownAwardNetAmount,
+                valueCode: "CSGP",
+                valueType: "Canada Grant",
+              },
+              {
+                awardEligibility: true,
+                valueAmount: testFinalAwardNetAmount.roundDownAwardNetAmount,
+                valueCode: "CSGD",
+                valueType: "Canada Grant",
+              },
+              {
+                awardEligibility: true,
+                valueAmount: testFinalAwardNetAmount.roundDownAwardNetAmount,
+                valueCode: "CSPT",
+                valueType: "Canada Grant",
+              },
+              {
+                awardEligibility: true,
+                valueAmount: testFinalAwardNetAmount.roundDownAwardNetAmount,
+                valueCode: "BCAG",
+                valueType: "BC Grant",
+              },
+              {
+                awardEligibility: true,
+                valueAmount: testFinalAwardNetAmount.roundDownAwardNetAmount,
+                valueCode: "SBSD",
+                valueType: "BC Grant",
+              },
+            ],
+          },
+        ]);
+      });
+    }
+  });
+
+  describe(
```

Comment:
Is this test not similar to the previous one? Can you please clarify the different goals between both?
Just double-checking if the first one would not be enough to cover the modification.